packages/cubejs-server-core/core/index.d.ts

```diff
@@ -39,7 +39,7 @@ export interface CreateOptions {
   updateCompilerCacheKeepAlive?: boolean;
   telemetry?: boolean;
   allowUngroupedWithoutPrimaryKey?: boolean;
-  orchestratorOptions?: OrchestratorOptions;
+  orchestratorOptions?: OrchestratorOptions | (() => OrchestratorOptions);
```

Comment:
I found inside source code that it passes `context`, it thinks it should be

```
orchestratorOptions?: OrchestratorOptions | ((context: RequestContext) => OrchestratorOptions);
```

cC @vasilev-alex @paveltiunov

Reply:
Good catch, I’ve updated the PR. 

Here is where it passes context:
https://github.com/cube-js/cube.js/blob/1c354df8c5e515614910729a40fe52c2e29aef0f/packages/cubejs-server-core/core/index.js#L490